README.md

```diff
@@ -0,0 +1,155 @@
+
+
+
+
+
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+
+# 📖 Blog App <a name="about-project"></a>
+
+> The Blog app is a classic example of a blog website. It a fully functional website that will show the list of posts and empower readers to interact with them by adding comments and liking posts.
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Ruby on Rails</summary>
+  <ul>
+    <li><a href="https://ruby-doc.org/core-3.1.2/">Ruby</a></li>
+  </ul>
+    <ul>
+    <li><a href="https://ruby-doc.org/core-3.1.2/">Rails</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Create blog plost**
+
+- **Comment on blog post**
+
+- **Display list of comments**
+
+p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Code Editor (Vs Code)
+
+- Terminal
+
+- Install ruby on your computer
+
+- Install rails on your computer
+
+### Setup
+
+Clone this repository by using the command line to your desired folder:  
+
+```sh
+git clone https://github.com/jodavid444/Blog_Application.git
+
+cd Blog_Application
+```
+
+### Install
+
+Install this project with:
+
+```sh
+cd Blog_Application
+ bundle install (install all dependencies)
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+rails server
+```
+
+### Test
+
+```sh
+  rspec
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## Authors <a name="authors"></a>
+
+👤 **Joseph David**
+
+- GitHub: [@jodavid444](https://github.com/jodavid444)
+
+- Twitter: [@jodavid444dave](https://twitter.com/jodavid444dave)
+
+- LinkedIn: [@joseph-david](https://www.linkedin.com/in/joseph-david-/)
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- **Improve styles of the project**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/jodavid444/blog-app/issues)
```

Comment:
- [ ] @jodavid444 Great work on your documentation, however, the URL provided is the wrong one kindly check the project name in the URL it doesn't match this project name.

Reply:
Thank you for spending quality time reviewing my project, I have fix the broken link, kindly review it again @danmainah